docs/intro/getting-started-with-sphinx.rst

```diff
@@ -96,13 +96,13 @@ We support this natively on Read the Docs, and you can do it locally:
 
 .. prompt:: bash $
 
-    pip install recommonmark
+    pip install myst-parser
 
 Then in your ``conf.py``:
 
 .. code-block:: python
 
-   extensions = ['recommonmark']
+   extensions = ['myst-parser']
 
 .. warning:: Markdown doesn't support a lot of the features of Sphinx,
```

Comment:
I think we can remove or update this warning, myst does support directives and roles.

Reply:
Let me remove the warning and update the PR.

Reply:
@stsewd on second thought, I think it's best to just update the warning. As the warning also mentioned to use reStructuredText instead of markdown with a blog post. So I think it would benefit anyone waiting to write documentation but is confused between markdown and reStructuredText